suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -58,24 +60,14 @@ const TodayHeaderCell = ({
         Number,
     );
 
-    const hasPriceIncreased = fiatTotalHistoryNumeric < fiatTotalActualNumeric;
-    const isPercentDifferenceVisible = fiatTotalHistoryNumeric !== fiatTotalActualNumeric;
-
-    const percentageDifference = Math.abs(
-        Math.round(
-            ((fiatTotalActualNumeric - fiatTotalHistoryNumeric) / fiatTotalHistoryNumeric) * 100,
-        ),
-    );
-
     return (
         <Text variant="hint" color="gray600">
             Today{' '}
-            {isPercentDifferenceVisible && (
-                <Text variant="hint" color={hasPriceIncreased ? 'green' : 'red'}>
-                    <SignValueFormatter value={hasPriceIncreased ? 'positive' : 'negative'} />
-                    {`${percentageDifference}`}%
-                </Text>
-            )}
+            <PercentageDifferenceFormatter
+                value={fiatTotalActualNumeric}
```

Comment:
Shouldn't be the hisotric value as first `value` and the actual value `secondValue`? Seems more intuitive to me like that. Just a thought.

Reply:
Yeah, good point :) https://github.com/trezor/trezor-suite/pull/7586/commits/65e36505040d1608298e27e5713807ab154c90b3